python-package/lightgbm/dask.py

```diff
@@ -109,8 +105,8 @@ def _find_ports_for_workers(client: Client, worker_addresses: Iterable[str], loc
 def _concat(seq):
     if isinstance(seq[0], np.ndarray):
         return np.concatenate(seq, axis=0)
-    elif isinstance(seq[0], (pd.DataFrame, pd.Series)):
-        return pd.concat(seq, axis=0)
+    elif isinstance(seq[0], (DataFrame, Series)):
+        return concat(seq, axis=0)
```

Comment:
I'm totally agree! But I think we can make import aliases in `compat.py` and then import like `from .compat import pd_DataFrame`. Otherwise in case of identical names it will be confusing to have 
```
from dask import DataFrame
from pandas import DataFrame
```
in `compat.py`.

I'm going to rename only Dask imports in this PR to not overcomplicate review. pandas will be done in a follow-up PR. Do you agree?

Reply:
yes sounds good, thank you